packages/graphql-client/.eslintignore

```diff
@@ -0,0 +1,15 @@
+rollup.config.js
+rollup-common.js
+rollup-umd.config.js
+.eslintrc.js
+node_modules/
+dist/
+dist-umd/
+schema.json
+
+graphql.schema.json
+src/types/apiTypes.ts
+**/*.d.ts
+src/clientPlus/graphql/operations.ts
+codegen.ts
+transform-graphql-jest-28-shim.js
```

Comment:
Seems like we might be ignoring some files that don't exist here?

Reply:
Fixed!